notebook_examples/example.ipynb

```diff
@@ -6,15 +6,67 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Import altair first\n",
-    "import altair as alt\n",
     "from onecodex.notebooks.report import title\n",
-    "import warnings\n",
+    "title(\"Example Report\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Introduction\n",
+    "\n",
+    "This notebook contains all of the code from the corresponding post on the [One Codex Blog](https://www.onecodex.com/blog/2019/05/01/onecodex-library-for-data-viz/). These snippets are exactly what are in the blog post, and let you perfectly reproduce those figures.\n",
```

Comment:
I think we already have this notebook in a separate file (`notebooks_demo.ipynb`)